README.md

```diff
@@ -22,13 +22,13 @@ The following table provides a detailed list of all models materialized within t
  
 | **model** | **description** |
 |-----------|-----------------|
-| [recharge__billing_history](https://fivetran.github.io/dbt_recharge/#!/model/model.recharge.recharge__billing_history) | Each record represents an order, enriched with metrics about related charges and line items. |
+| [recharge__billing_history](https://fivetran.github.io/dbt_recharge/#!/model/model.recharge.recharge__billing_history) | Each record represents an order, enriched with metrics about related charges and line items. | 
```

Comment:
Small wording update request. Can you clarify that this will aggregate line items as this is at the billing level.